From 3df90cefe5d60fd8f3a8c54cb9a3c8e03c0e66c1 Mon Sep 17 00:00:00 2001
From: Yan Zhang <cyanogenoid@cyanogenoid.com>
Date: Thu, 30 Apr 2015 11:35:13 +0100
Subject: [PATCH] Disable debug in bigint.c

---
 avr-crypto-lib/bigint/bigint.c | 4 +---
 1 file changed, 1 insertion(+), 3 deletions(-)

diff --git a/avr-crypto-lib/bigint/bigint.c b/avr-crypto-lib/bigint/bigint.c
index 6d77cdf..efc2223 100644
--- a/avr-crypto-lib/bigint/bigint.c
+++ b/avr-crypto-lib/bigint/bigint.c
@@ -42,8 +42,6 @@
 #define ALLOC_BIGINT_WORDS(var,words)                                      \
     bigint_word_t *(var) = malloc((words) * sizeof(bigint_word_t));        \
     if (!(var)) {                                                          \
-        puts_P(PSTR("\n\nDBG: OOM ERROR (in arithmeics)!\n"));             \
-        uart0_flush();                                                     \
         for(;;)                                                            \
             ;                                                              \
     }
@@ -57,7 +55,7 @@
 
 #endif
 
-#define DEBUG 1
+#define DEBUG 0
 
 #if DEBUG
 #include "cli.h"
-- 
1.9.1

